media-gfx/pngcrush/pngcrush-1.8.9.recipe

```diff
@@ -10,8 +10,8 @@ COPYRIGHT="1998-2002, 2006-2016 Glenn Randers-Pehrson
 	2005 Greg Roelofs"
 LICENSE="LibPNG"
 REVISION="1"
-SOURCE_URI="http://downloads.sf.net/pmt/pngcrush-$portVersion-nolib.tar.xz"
-CHECKSUM_SHA256="e20f8d823c778815d788d1f1d0275d98d76774695dca4f2d273f7f9ddf0a3bcb"
+SOURCE_URI="http://downloads.sf.net/pmt/pngcrush-$portVersion-nolib.tar.gz"
```

Comment:
I'm switching from gzip to xz to allow x86_gcc2 _non hybrid_ to build this (since xz is not available on x86_gcc2 _non hybrid_).
May I merge this as is, or should I revert the switch (and put the SHA256 of the tar.xz) ?